taca/utils/misc.py

```diff
@@ -109,6 +110,25 @@ def days_old(date, date_format="%y%m%d"):
         return None
     return time_dif.days
 
+
+def to_seconds(days=None, hours=None):
+    """ Convert given day/hours to seconds and return
+
+        :param int days: days to be converted
+        :param int hours: hours to be converted
+    """
+    #only either days or hours should be speciefied, but atleast one should be specified
+    if days and hours:
+        raise SystemExit('Both "days" and "hours" were given, use only either of them')
+    elif not days and not hours:
```

Comment:
Setting default to **0** is the same as setting them to `None`, isn't it ? :eyes: 

But if you meant **1** for both arguments, than that would also be a problem in the previous check, as `if days and hours` will always be satisfied. I can change the exit message to `raise SystemExit('provide either "days" or "hours, it should be atleast 1 hour"')` ?